app/models/payment.rb

```diff
@@ -0,0 +1,10 @@
+class Payment < ApplicationRecord
+    # Associations
+    belongs_to :appointment, optional: true
```

Comment:
Add a todo add top to mention that we need to remove optional true in future:

`TODO: Remove optional: true once appointment model is added`

Reply:
Done
```
# TODO: Remove optional: true once appointment model is added
class Payment < ApplicationRecord
    # Associations
    belongs_to :appointment, optional: true
  
    # Money-rails setup
    monetize :amount_cents, with_model_currency: :amount_currency

    # Validations
    validates :amount_cents, presence: true, numericality: { greater_than: 0 }
    validates :stripe_id, presence: true, uniqueness: true
    validates :appointment_id, presence: true, uniqueness: true  
  end
```